build/build.go

```diff
@@ -26,3 +26,13 @@ func Runfile() string {
 	// return converted output value
 	return output
 }
+
+// Native function to call Linux command
+func Native() string {
+	cmd, err := exec.Command("/bin/sh", "/Users/richard/Desktop/Go/jekyll-server-build/build/echo.sh").Output()
+	if err != nil {
+		fmt.Printf("error %s", err)
```

Comment:
In general, libraries should never print (and I know this is just a piece of constructive play, but it's a good opportunity to make the point). If you find yourself _writing_ `if err != nil`, then you should consider whether the function needs to return an error value. For example, we might change the signature of Native() to:

```go
func Native() (string, error) {
```

and we might handle this error with something like:

```go
return "", fmt.Errorf("failed to run %q: %w", cmdLine, err)
```

where `cmdLine` is the string containing the command we tried to run. (Why `%w` instead of `%s`? See [Working with Errors in Go 1.13](https://blog.golang.org/go1.13-errors))

Reply:
Noted, I think many of the videos I watched on PluralSight have influenced my approach, which is understandable since those videos were oriented toward understanding general syntax.